applications/StructuralMechanicsApplication/custom_conditions/displacement_control_condition.cpp

```diff
@@ -104,6 +104,45 @@ Condition::Pointer DisplacementControlCondition::Clone (
 /***********************************************************************************/
 /***********************************************************************************/
 
+DisplacementControlCondition::Array1DComponentType* DisplacementControlCondition::GetDisplacementInDirection()
+{
+    double numerical_limit = std::numeric_limits<double>::epsilon();
+    array_1d<double, 3 > point_load = ZeroVector(3);
+    if( this->Has( POINT_LOAD ) )
+        noalias(point_load) = this->GetValue( POINT_LOAD );
+
+    if (abs(point_load(0)) > numerical_limit) // prescribed displacement in x direction
```

Comment:
NOOOOOOO
![imagen](https://user-images.githubusercontent.com/19991680/35767644-8a9c6856-08f0-11e8-9944-e72c6c1db7b3.png)

abs, instead of std::abs returns an integer

Reply:
Oh :man_facepalming: sorry